test/e2e/specs/editor/various/patterns.spec.js

```diff
@@ -254,22 +279,15 @@ test.describe( 'Synced pattern', () => {
 		// Go back to the post.
 		await editorTopBar.getByRole( 'button', { name: 'Back' } ).click();
 
-		await expect.poll( editor.getBlocks ).toEqual( [
-			{
-				name: 'core/block',
-				attributes: { ref: id },
-				innerBlocks: [
-					{
-						name: 'core/paragraph',
-						attributes: {
-							content: 'Einen Guten Tag!',
-							dropCap: false,
-						},
-						innerBlocks: [],
-					},
-				],
-			},
-		] );
+		const [ syncedPattern ] = await editor.getBlocks();
+		const patternRecord = await getPatternRecord(
```

Comment:
Nit: Same here. I don't think we need to retest this. We can just remove this.

Reply:
Sorry, I totally missed [the previous comment](https://github.com/WordPress/gutenberg/pull/60721#discussion_r1584042125). I've modified the logic to match what you suggested (I hope 😄 ).
- Checking that the "core/block" is present and check the frontend in the first test: [link](https://github.com/WordPress/gutenberg/pull/60721/commits/d80aba74b6f9f4eb55cc0209e4e8f55e9850b72b).
- Checking the frontend also when we modify the pattern: [link](https://github.com/WordPress/gutenberg/pull/60721/commits/6b97321200cf7f0f7c225e2738fa2c1d309c4b8f).
- Just check that the "core/block" is present in the rest of the tests: [link](https://github.com/WordPress/gutenberg/pull/60721/commits/c122b7991898c00b8d85fa59d174f3950aaca1cf).

Let me know if that's not what you had in mind 🙂

Reply:
Yes! Thank you for improving the tests ❤️!

We could also check the editor but I don't think it's necessary. It's basically just calling `getBlocks()` and we'd just be testing implementation details. I think the tests now should cover almost everything.